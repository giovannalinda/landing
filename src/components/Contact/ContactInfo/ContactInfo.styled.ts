import styled from 'styled-components'
import { theme } from '~/styles'

export const ButtonGroup = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-top: ${theme.space[5]};
`
