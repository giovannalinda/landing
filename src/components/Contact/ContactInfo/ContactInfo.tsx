import Link from 'next/link'

import { ImDribbble, ImGithub, ImLinkedin2 } from 'react-icons/im'
import { BsInstagram, BsMedium } from 'react-icons/bs'

import { Button } from '~/components'
import { ContactList } from './ContactList'

import * as S from './ContactInfo.styled'

export function ContactInfo() {
  return (
    <>
      <h1 style={{ textAlign: 'center' }}>Get In Touch</h1>

      <ContactList />

      <S.ButtonGroup>
        <Link href='https://dribbble.com/giovannalinda' target='_blank'>
          <Button variant='outlined' size='md' rounded>
            <ImDribbble size={20} />
          </Button>
        </Link>
        <Link href='https://github.com/giovannalinda' target='_blank'>
          <Button variant='outlined' size='md' rounded>
            <ImGithub size={20} />
          </Button>
        </Link>
        <Link href='https://www.linkedin.com/in/giovannalinda' target='_blank'>
          <Button variant='outlined' size='md' rounded>
            <ImLinkedin2 size={20} />
          </Button>
        </Link>
        <Link href='https://www.instagram.com/_gripada' target='_blank'>
          <Button size='md' variant='outlined' rounded>
            <BsInstagram size={20} />
          </Button>
        </Link>
        <Link href='https://medium.com/@eugiovannasouza' target='_blank'>
          <Button size='md' variant='outlined' rounded>
            <BsMedium size={20} />
          </Button>
        </Link>
      </S.ButtonGroup>
    </>
  )
}
