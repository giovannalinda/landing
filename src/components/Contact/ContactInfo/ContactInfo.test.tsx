import { render } from '~/__helpers__/app-tests'
import { ContactInfo } from './ContactInfo'

describe('ContactInfo', () => {
  it('should match snapshot', () => {
    const { container } = render(<ContactInfo />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
