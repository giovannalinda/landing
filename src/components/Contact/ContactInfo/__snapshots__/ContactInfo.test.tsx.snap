// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInfo should match snapshot 1`] = `
<h1>
  Get In Touch
</h1>
`;
